cmd/thriftrw-list-deps/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+
+	"go.uber.org/thriftrw/compile"
+)
+
+type args struct {
+	ThriftFile string `positional-arg-name:"thrift-file"`
+}
+
+var opts struct {
+	Relative string `long:"relative" description:"Output paths will be relative to this directory"`
+	Args     args   `positional-args:"yes" required:"yes"`
+}
+
+func do(relative string, inputThriftFile string, output io.Writer) error {
+	module, err := compile.Compile(inputThriftFile)
+	if err != nil {
+		return err
+	}
+
+	for _, included := range module.Includes {
+		outputFilePath := included.Module.ThriftPath
+		if relative != "" {
+			outputFilePath, err = filepath.Rel(relative, outputFilePath)
+			if err != nil {
+				return err
```

Comment:
The error message should include more context.

    fmt.Errorf("cannot make %q relative to %q: %v", outputFilePath, relative, err)